tests/integration_tests/functional/test_max_devices.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests scenario for adding the maximum number of devices to a microVM."""
+
+import host_tools.network as net_tools
+
+def test_attach_maximum_devices(test_microvm_with_ssh, network_config):
+    """Test attaching maximum number of devices to the microVM."""
+    test_microvm = test_microvm_with_ssh
+    test_microvm.spawn()
+
+    # Set up a basic microVM.
+    test_microvm.basic_config()
+
+    # IRQs are available from 5 to 23, so the maximum number of devices
+    # supported at the same time is 19.
+    # Adding 19 devices (including the rootfs, which has already been
+    # configured in the `basic_config()`function) should work.
+    guest_ips = []
+    for i in range(18):
```

Comment:
Let's have this `18` as a constant outside the tests functions.

Reply:
Ack